feat(changelog): elastic-metal-added-new-public-bandwidth-increasing-opt 2024-09-26 (#3759)
scaleway/docs-content: changelog/september2024/2024-09-26-elastic-metal-added-new-public-bandwidth-increasing-opt.mdx

---
title: New public bandwidth increasing option!
status: added
author:

>     fullname: 'Join the #[elastic-metal] channel on Slack.'

Reviewer: ```suggestion
    fullname: 'Join the #elastic-metal channel on Slack.'
```

    url: 'https://slack.scaleway.com'
date: 2024-09-26
category: bare-metal
product: elastic-metal
---

> On compliant Elastic Metal offers, you can now increase the **public bandwidth** as you wish either through console or through API.

Reviewer: ```suggestion
On compliant Elastic Metal offers, you can now increase the **public bandwidth** as you wish through the console or API.
```

The eligible offers where you can increase the public bandwidth up to 10 Gbps are:
* EM-B220E-NVME
* EM-B320E-NVME
* EM-B420E-NVME
* EM-B520E-NVME
* EM-I115E-NVME
* EM-L220E-NVME
* EM-L420E-NVME

> To activate it, go to the [Elastic Metal server section](https://console.scaleway.com/elastic-metal/servers) of the Scaleway console and click **Settings** > **Public bandwidth** > **Manage bandwidth**

Reviewer: ```suggestion
To activate it, go to the [Elastic Metal server section](https://console.scaleway.com/elastic-metal/servers) of the Scaleway console and click **Settings** > **Public bandwidth** > **Manage bandwidth**.
```

---

> title: 🌐 New public bandwidth increasing option!

Reviewer 2: ```suggestion
title: New public bandwidth increasing option!
```

I removed the emoji from the title to avoid issues with the console

status: added
author:
    fullname: 'Join the #[elastic-metal] channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-09-26
category: bare-metal
product: elastic-metal
---

On compliant Elastic Metal offers, you can now increase the **public bandwidth** as you wish either through console or through API.
The eligible offers where you can increase the public bandwidth up to 10 Gbps are:
* EM-B220E-NVME
* EM-B320E-NVME
* EM-B420E-NVME
* EM-B520E-NVME
* EM-I115E-NVME
* EM-L220E-NVME
* EM-L420E-NVME

> To activate it, go to your Elastic Metal server administration and then go to **Settings** > **Public bandwidth** > **Manage bandwidth**

Reviewer 2: ```suggestion
To activate it, go to the [Elastic Metal server section](https://console.scaleway.com/elastic-metal/servers) of the Scaleway console and click **Settings** > **Public bandwidth** > **Manage bandwidth**
```

> <Steps>
>   <StepButton to=https://console.scaleway.com/elastic-metal/servers/>Manage my Elastic Metal servers</StepButton>
> </Steps>

Reviewer 2: ```suggestion
```

> *Prices of this option are available [here](https://www.scaleway.com/en/pricing/elastic-metal/#network).*

Reviewer 2: ```suggestion
Prices of this option are available [here](https://www.scaleway.com/en/pricing/elastic-metal/#network).
```
